QuoteGeneration/pccs/container/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:20.04
+
+# DCAP version (github repo branch, tag or commit hash)
+ARG DCAP_VERSION=DCAP_1.10.3
+
+ARG USER=pccs
+ARG UID=65333
+
+# update and install packages
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get update -yq && \
+    apt-get upgrade -yq && \
+    apt-get install -yq --no-install-recommends \
+            build-essential \
+            ca-certificates \
+            curl \
+            git \
+            zip
+
+# install node.js
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends nodejs
+
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# clone DCAP repo
+RUN git clone https://github.com/intel/SGXDataCenterAttestationPrimitives.git -b ${DCAP_VERSION} && \
+    rm -rf /SGXDataCenterAttestationPrimitives/.git/
+
+# set PWD to PCKCertSelection dir
+WORKDIR /SGXDataCenterAttestationPrimitives/tools/PCKCertSelection/
+
+# build libPCKCertSelection library and copy to lib folder
+RUN make && \
+    mkdir -p ../../QuoteGeneration/pccs/lib && \
+    cp ./out/libPCKCertSelection.so ../../QuoteGeneration/pccs/lib/ && \
+    make clean
+
+# set PWD to PCCS dir
+WORKDIR /SGXDataCenterAttestationPrimitives/QuoteGeneration/pccs/
+
```

Comment:
I suggest docker multi-stage build (https://docs.docker.com/develop/develop-images/multistage-build/) to minimize dependencies at runtime.

Reply:
Fixed.